### ISSUE TITLE
im not able to install anonsurf into my kali linux 2017.2

### ISSUE THREAD

**Issue by sam7440** [Und3rf10w/kali-anonsurf#47]

**Comment by sam7440** [comment]
some please help me about my issue

**Comment by mkpmanish** [comment]
same issue with me.......libservlet3.0-java not installed

```
E: Unable to locate package libservlet3.0-java
E: Couldn't find any package by glob 'libservlet3.0-java'
E: Couldn't find any package by regex 'libservlet3.0-java'
--2017-10-14 19:30:37--  http://ftp.us.debian.org/debian/pool/main/j/jetty8/libjetty8-java_8.1.16-4_all.deb
Resolving ftp.us.debian.org (ftp.us.debian.org)... 128.61.240.89, 128.30.2.26, 208.80.154.15, ...
Connecting to ftp.us.debian.org (ftp.us.debian.org)|128.61.240.89|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1666404 (1.6M) [application/octet-stream]
Saving to: ‘libjetty8-java_8.1.16-4_all.deb’

libjetty8-java_8.1.16-4_all.deb       100%[=======================================================================>]   1.59M   364KB/s    in 4.5s    

2017-10-14 19:30:42 (364 KB/s) - ‘libjetty8-java_8.1.16-4_all.deb’ saved [1666404/1666404]

Selecting previously unselected package libjetty8-java.
(Reading database ... 372747 files and directories currently installed.)
Preparing to unpack libjetty8-java_8.1.16-4_all.deb ...
Unpacking libjetty8-java (8.1.16-4) ...
dpkg: dependency problems prevent configuration of libjetty8-java:
 libjetty8-java depends on libservlet3.0-java; however:
  Package libservlet3.0-java is not installed.

dpkg: error processing package libjetty8-java (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 libjetty8-java

```

**Comment by mkpmanish** [comment]
Well was able to resolve issue , try below steps and see if it helps ( catch is to add the repo that contain libservlet3.0-java ):-

adding repo of libservlet3.0-java 👍 
`
echo "deb http://ftp.de.debian.org/debian sid main" >> /etc/apt/sources.list
`
do a repo refresh/update

`sudo apt-get update`
 
then try installing from `./installer.sh`

if it still fails try using `apt-get install libservlet3.0-java` , we can also check by `apt-cache search libservlet` to see if you find in repo.

![image](https://user-images.githubusercontent.com/10396257/31576592-7827d534-b11b-11e7-9f58-535e8ceb263e.png)

**Comment by sam7440** [comment]
yes bro me too found it somehow and thanks for your concern appreciate your work

**Comment by fme69** [comment]
hey i m having a different issue its giving errors 
--2017-12-30 07:17:44--  https://geti2p.net/_static/i2p-debian-repo.key.asc
Resolving geti2p.net (geti2p.net)... 91.143.92.136, 2a02:180:a:65:2456:6542:1101:1010
Connecting to geti2p.net (geti2p.net)|91.143.92.136|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 15200 (15K) [text/plain]
Saving to: ‘/tmp/i2p-debian-repo.key.asc’

/tmp/i2p-debian-rep 100%[===================>]  14.84K  89.0KB/s    in 0.2s    

2017-12-30 07:17:45 (89.0 KB/s) - ‘/tmp/i2p-debian-repo.key.asc’ saved [15200/15200]

OK
Ign:1 http://dl.google.com/linux/chrome/deb stable InRelease
Get:2 http://security.ubuntu.com/ubuntu artful-security InRelease [78.6 kB]    
Hit:3 http://deb.i2p2.no unstable InRelease                                    
Hit:4 http://dl.google.com/linux/chrome/deb stable Release                     
Hit:5 http://us.archive.ubuntu.com/ubuntu artful InRelease                     
Get:7 http://us.archive.ubuntu.com/ubuntu artful-updates InRelease [78.6 kB]   
Fetched 157 kB in 1s (106 kB/s)                                
Reading package lists... Done
Reading package lists... Done
Building dependency tree       
Reading state information... Done
i2p-keyring is already the newest version (2016.10.01).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
secure-delete is already the newest version (3.1-6ubuntu1).
tor is already the newest version (0.3.0.10-1).
i2p is already the newest version (0.9.32-1ubuntu1).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
dpkg-deb: error: control directory has bad permissions 700 (must be >=0755 and <=0775)
dpkg: error: cannot access archive 'kali-anonsurf.deb': No such file or directory
Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
dpkg: error: cannot access archive 'kali-anonsurf.deb': No such file or directory
